src/push.js

```diff
@@ -25,23 +25,23 @@ const TEMP_GCM_URL = 'https://gcm-http.googleapis.com/gcm';
 /**
  * URL safe Base64 encoder
  * @param  {Buffer} buffer The data to encode
- * @return {String}
+ * @return {String} URL safe base 64 encoded string
  */
 function ub64(buffer) {
   return buffer.toString('base64').replace(/\+/g, '-')
     .replace(/\//g, '_')
-    .replace(/=/g, '');
-};
+    .replace(/\=/g, '');
```

Comment:
`=` doesn't need to be escaped. It's not special on it's own, only as part of the sequence `(?=`